src/app-layout/visual-refresh/context.tsx

```diff
@@ -33,15 +33,15 @@ import { useContainerQuery } from '@cloudscape-design/component-toolkit';
 import useBackgroundOverlap from './use-background-overlap';
 
 interface AppLayoutInternals extends AppLayoutProps {
-  activeDrawerId?: string | null;
+  activeDrawerId?: string | undefined;
```

Comment:
There was a discrepancy between refresh and classic version. Classic used `undefined` to notify "no active drawer" state and refresh used "null".

To avoid surprises on consumers end when they implement dynamic switch to refresh, we should use a consistent approach

Reply:
Agreed they should be consistent. In the API proposal, we actually agreed on changing both to use `null` instead of `undefined` since there is a valid use case for no active drawer open. So this is fine for now, but planning on changing both to use `null` in a future PR (there are also a few other API discrepancies between the two that I'll need to fix simultaneously)

Reply:
Fine by me either way as long as it is consistent